frame.go

```diff
@@ -87,6 +89,35 @@ func (f *Frame) DataBytes() ([]byte, error) {
 
 }
 
+// SetDataBytes fills the frame data with given raw data. The data must match the pixel format and the dimensions of the frame.
+func (f *Frame) SetDataBytes(data []byte) (err error) {
+
+	// Get the expected buffer length and check data. If data does not have the correct length, we stop and return an
+	// error. Otherwise, pointers will be used to access unexpected memory addresses.
+	expectedBufferLength := C.av_image_get_buffer_size(C.enum_AVPixelFormat(f.c.format), f.c.width, f.c.height, 1)
+	if expectedBufferLength < 0 {
+		return newError(expectedBufferLength)
+	}
+	if int(expectedBufferLength) != len(data) {
+		return fmt.Errorf("astiav: invalid data length of %d for frame. Expected %d", len(data), int(expectedBufferLength))
```

Comment:
Could you change the error to:

`fmt.Errorf("astiav: invalid buffer size (expected %d, got %d)", int(bufferSize), len(data))`